modules/social_features/social_follow_taxonomy/modules/social_follow_tag/social_follow_tag.install

```diff
@@ -335,3 +335,93 @@ function social_follow_tag_update_11301(): void {
     $config->save();
   }
 }
+
+/**
+ * Creates a new block content type.
+ */
+function social_follow_tag_update_11401(): void {
```

Comment:
@rollki Not sure if we need to change update number to 115xx since we move it to a new milestone.

Reply:
Yea, I think we ahve to change

Reply:
Fixed.

Reply:
Thanks!